### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by abel-tefera**
### Summary

- Initialized react with CRA, and installed react-router-dom and tailwindcss
- Created the components for book state, book list and categories
- Used react router in App.js for the different routes

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @abel-tefera :Wave, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly remove the Live Demo section from the README.md file. The Live Demo section should be present in the README if you are going to deploy the project. Since you won't be deploying the project, I would advise removing it. 👍 😎

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      <Book key={i} title={title} category={category} author={author} />
+    ))}
+    <form className="flex flex-col">
+      <div className="text-xl">Add New Book</div>
+      <div className="flex flex-row">
+        <input type="text" placeholder="Book Title" />
+        <input type="text" placeholder="Author" />
+        <button type="submit">Add Book</button>
+      </div>
+    </form>
```
Comment (DammyShittu) [inline src/components/Books.js]: - [ ] This component should be its own component. Kindly move this to a new component and import it to this component.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @abel-tefera,

## Highlights
✔️ Routes and render of components

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @abel-tefera 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**